cases-SEP31/send.test.js

```diff
@@ -30,26 +30,35 @@ describe("POST /send", () => {
   });
 
   it("fails with no authentication", async () => {
-    const resp = await fetch(toml.DIRECT_PAYMENT_SERVER + "/send", {
-      method: "POST",
-      body: JSON.stringify({
-        amount: 100,
-      }),
-    });
-    expect(resp.status).toBe(401);
+    const { json, status, logs } = await loggableFetch(
+      toml.DIRECT_PAYMENT_SERVER + "/send",
+      {
+        method: "POST",
+        body: JSON.stringify({
```

Comment:
Can you specify `headers` in the body like this?

Reply:
absolutely not